### PR TITLE
Edits to root help page; reorder authors; add CoC

### PR DESCRIPTION
**Pull request by bpbond**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
```
Comment (leeyap) [inline CODE_OF_CONDUCT.md:1]: I think this is a really good thing to have! Thoughts on adding a line on what happens if a community leader violates the code?

**Review by leeyap — APPROVED** [review]
Looks good 👍

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (bpbond) [reply CODE_OF_CONDUCT.md:1]: That is spelled out in the link document, but sure, might be useful to have a sentence here too. Thanks

Comment (bpbond) [reply CODE_OF_CONDUCT.md:1]: Oh I misread your comment

Comment (bpbond) [reply CODE_OF_CONDUCT.md:1]: What _should_ happen? I've been looking through a [variety](https://www.eclipse.org/org/documents/Community_Code_of_Conduct.php) of [online](https://ropensci.org/code-of-conduct/) [codes](https://www.hiretechladies.com/code-of-conduct/) of [conduct](https://www.jenkins.io/project/conduct/) and not seeing this addressed anywhere.

Comment (bpbond) [reply CODE_OF_CONDUCT.md:1]: It's obvious, I think, that the code of conduct applies equally to leadership as it does to other contributors.

Comment (leeyap) [reply CODE_OF_CONDUCT.md:1]: True, my thought was that they wouldn't be part of a decision on consequences (which is obvious, I think) but also if there is a violation, do they lose certain leadership privileges? Depending on the severity of action, of course.

Comment (bpbond) [reply CODE_OF_CONDUCT.md:1]: @leeyap See text change I just pushed. Feedback welcome.

Comment (leeyap) [reply CODE_OF_CONDUCT.md:1]: Looks good, thanks!

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+This Code of Conduct applies within all community spaces, and also applies
+when an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail
+address, posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the project leads, Kalyn Dorheim and/or Ben Bond-Lamberty.
```
Comment (kdorheim) [inline CODE_OF_CONDUCT.md:62]: @bpbond should we include contact information?

**Review by kdorheim — APPROVED** [review]
Looks good @bpbond & @leeyap nice work! one minor suggestion

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (bpbond) [reply CODE_OF_CONDUCT.md:62]: Sure; what? Emails?

Comment (bpbond) [reply CODE_OF_CONDUCT.md]: Added emails in [c44c8cf](https://github.com/JGCRI/hector/pull/592/commits/c44c8cf5eb8a425af351757ee803071e0a85d38a)

**Pull request merged**